# basic modes
uint8 PX4_MODE_STANDBY = 0
uint8 PX4_MODE_STARTING = 1
uint8 PX4_MODE_FINISHED = 2
uint8 PX4_MODE_STOPPED = 2

uint8 base_mode # filled by MAV_MODE enum value or 0 if custom_mode != ''
string custom_mode # string mode representation or integer
---
bool mode_sent
